[MTM-60271] OAI-Secure tokens generated in BODY will be limited based… (#2358)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10-20-525-0-limit-number-of-tokens-generated-in-body-based-on-session-configuration.md

---
date: ""

> title: Limit number of tokens generated in body based on session configuration

Reviewer: ```suggestion
title: Limited number of tokens generated in body based on the session configuration
```

Reviewer: Here it says "in body", below it says "in API". Phrasing should be consistent. Also, should it be mentioned that tokens generated by cookies have already be limited?

Author: Imo I would rather skip this information. In the case of tokens generated in the cookies, tokens are not limited but rather sessions, so to not complicate this release notes, I will skip this information. This is already in the documentation described.

> title: Limited number of tokens generated in body based on the session configuration

Reviewer: ```suggestion
title: Limited number of tokens generated in the body based on the session configuration
```

> title: Limited number of tokens generated in the body based on the session configuration

Reviewer: ```suggestion
title: Number of tokens generated in the body is now limited in the session configuration
```

> title: Number of tokens generated in the body is now limited in the session configuration

Reviewer: ```suggestion
title: Number of tokens generated in the JSON body is now limited
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-60271
version: 10.20.525.0
---

> To enhance OAI-Secure token management, the number of tokens generated in the JSON body will be limited to the maximum configured in the sessionConfiguration.

Reviewer: ```suggestion
To enhance the OAI-Secure token management, the number of tokens generated in the JSON body will be limited to the maximum configured in the session configuration.
```

> The maximum number of OAI-Secure Tokens generated by the "/tenant/oauth/token" API will be limited according to the `Maximum parallel sessions per user` in session configuration.

Reviewer: ```suggestion
The maximum number of OAI-Secure tokens generated by the `/tenant/oauth/token` API is now limited according to the `Maximum parallel sessions per user` parameter in the session configuration.
```

> The maximum number of OAI-Secure tokens generated in body by the `/tenant/oauth/token` API is now limited according to the `Maximum parallel sessions per user` parameter in the session configuration.

Reviewer: ```suggestion
The maximum number of OAI-Secure tokens generated in the body by the `/tenant/oauth/token` API is now limited according to the `Maximum parallel sessions per user` parameter in the session configuration.
```
